index.css

```diff
@@ -463,3 +467,68 @@ footer form .message-input {
   top: 15px;
   bottom: 15px;
 }
+
+@media screen and (min-width: 768px) {
+  .desktop-only {
+    display: inline;
+  }
+}
+
```

Comment:
There is no default screen version, desktop version should be a default version, not required media query